docs/jsonapi-spec.textile

```diff
@@ -408,12 +502,33 @@ h4(#commandFindOneAndUpdateOptions). findOneAndUpdate Command Options
 | @returnDocument@ | String Enum | Specifies which document to perform the projection on. If @"before"@ the projection is performed on the document before the update is applied, if @"after"@ the document projection is from the document after the update. Defaults to @"before"@.|
 | @upsert@ | Boolean | When @true@ if no documents match the @filter@ clause the command will create a new _empty_ document and apply the @update@ clause to the empty document. If the @_id@ field is included in the @filter@ the new document will use this @_id@, otherwise a random value will be used see "Upsert Considerations":#considerationsUpsert for details. When false the command will only update a document if one matches the filter. Defaults to @false@.|
 
+h4(#commandFindOneAndUpdateResponse). findOneAndUpdate Command Response
+
+|_. Response Element |_. Description |
+| @data@ | Present with fields : @data@ only |
```

Comment:
again `@docs@`? what doc is returned, unclear?

Reply:
fixed docs, the projection is controlled by the @returnDocument@ in the options, will mention that